### PR TITLE
chore(website): Update main README

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

**Review by erezrokah — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <h1 align="center"><img alt="cloudquery logo" width=75% src="https://github.com/cloudquery/cloudquery/raw/main/cli/docs/images/logo.png"/></h1>
 <!-- markdownlint-enable MD033 -->
 
-CloudQuery is an open-source,
-high-performance data integration platform for security and infrastructure teams.
+CloudQuery is an [open-source](https://github.com/cloudquery/cloudquery),
+high-performance data integration framework built for developers.
```
Comment (hermanschaaf) [inline README.md:6]: Is it `framework` or `platform` now? The website has `platform` 😄 Personally I like `framework`

File: README.md [hunk 2]
```diff
@@ -15,18 +15,18 @@ See the **[Quickstart guide](https://www.cloudquery.io/docs/quickstart)** for in
 
 ## Why CloudQuery?
 
-- **Open source**: Easily extendable plugin architecture. Contribute to our official plugins or develop your own with the CloudQuery SDK.
-- **Blazing fast**: CloudQuery's concurrency system leverages the excellent Go concurrency model with light-weight Goroutines.
-- **Database agnostic**: CloudQuery can store your configuration in any supported destination such as a database, data lake, or streaming platform for further analysis.
-- **Pre-built queries**: CloudQuery maintains a rich set of SQL queries and Grafana dashboards for asset inventory, CSPMs, Security & Compliance, and Cost use-cases.
-- **Raw access to data**: Decouple data ingestion and have raw access to your data, with which you can build your own security stack and re-use
-  best-of-breed tools for querying (SQL,…), transformation (dbt,…), and visualization (Grafana, Preset, Metabase, PowerBI, …).
+- **Open source**: Extensible plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.
```
Comment (hermanschaaf) [inline README.md:18]: I read up about it earlier, and `extensible` is commonly used in the software world... I think it's fine, maybe we can add a Vale exception

File: README.md (hunk 1 above)
Comment (yevgenypats) [reply README.md:6]: yeah I think it is a bit interchangeable, we can lean more toward framework as we don't want to maintain the longtail of plugins.

**Review by hermanschaaf — COMMENTED** [review]
Looks good, we'll just need to add some vale exceptions

**Review by hermanschaaf — APPROVED** [review]

**Comment by erezrokah** [comment]
> ### This PR has missing tables

Ah this is new, but I think unrelated to this PR

**Pull request merged**